.github/workflows/test-sdks.yml

```diff
@@ -81,31 +81,10 @@ jobs:
           path: /Users/runner/work/sdk-test-data/sdk-test-data/eppo/build/reports/androidTests/connected/index.html
 
   test-node-server-sdk:
-    runs-on: ubuntu-latest
-    steps:
-      - name: 'Check out node server SDK'
-        uses: actions/checkout@v3
-        with:
-          repository: 'Eppo-exp/node-server-sdk'
-      - name: Use Node.js 16
-        uses: actions/setup-node@v1
-        with:
-          node-version: '16.x'
-      - uses: actions/cache@v2
-        with:
-          path: './node_modules'
-          key: ${{ runner.os }}-root-node-modules-${{ hashFiles('./yarn.lock') }}
-      - name: Install root dependencies
-        run: yarn --frozen-lockfile
-        working-directory: ./
-      - name: 'Set up GCP SDK for downloading test data'
-        uses: 'google-github-actions/setup-gcloud@v0'
-      - name: Install sdk dependencies
-        run: yarn --frozen-lockfile
-        working-directory: ./
-      - name: Run tests
-        run: yarn test:unit
-        working-directory: ./
+    uses: Eppo-exp/node-server-sdk/.github/workflows/lint-test-sdk.yml@tp/workflows/remote
```

Comment:
temporary branch spec _@tp/workflows/remote_ to be removed when SDK PRs are submitted.